site/frontend/src/pages/compare/compile/compile-page.vue

```diff
@@ -197,15 +215,33 @@ const urlParams = getUrlParams();
 const quickLinksKey = ref(0);
 const filter = ref(loadFilterFromUrl(urlParams, defaultCompileFilter));
 
+// Should we use the backend as the source of before/after data?
+const selfCompareBackend = computed(
+  () => comparesIdenticalCommits.value && filter.value.selfCompareBackend
```

Comment:
should we also check that we have profiling results for the 2 backends?

Reply:
Good point, added a check.